Provisioning Kubernetes Cluster

sowmya004 opened the pull request:
steps 3 to 19

Diego-Viero commented:
Why did you remove the ssh_keys folder? Do you handle authentication differently?
Be sure to also update the README file, in there there is still a section instructing to add the public key to the ssh_keys folder.

Diego-Viero commented:
The README clearly guides you through the setup, I could follow it without any error. Although the last step "visit the service at: 192.168.56.101:<NodePort>" didn't work for me.

When I check the nodes' status with `kubectl get nodes` I get:
ctrl     NotReady   control-plane   14m   v1.32.4
node-1   NotReady   <none>          13m   v1.32.4
node-2   NotReady   <none>          11m   v1.32.4

Maybe the NotReady part has something to do with it?

sowmya004 commented:
@Diego-Viero I had moved the .pub files to .gitignore. As it can cause confusion, I have added the folder back. We are still handling the authentication through public keys and the procedure mentioned in Readme file can be followed. 

As for 2nd issue - can you run this command as part of the kubectl setup and let me know if it works 
kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
(I have updated the readme accordingly)
